### PR TITLE
deployment

### PR DESCRIPTION
**Pull request by Florassss**
v 0.0

### CODE REVIEW

**Comment by Florassss** [comment]
I dont see a point using yaml; 
also the code makes the csv file.

**Comment by Florassss** [comment]
> @Florassss
> 
> Just call `n_name` `new_path` and remove the comment

what are you referring to? what comment

**Comment by Florassss** [comment]
> With `pathlib.Path` you can just say `filename.parent` which is more clear I think!

The reason I couldn't use pathlib for new path is the sub dir has space in the string. that's why I am replacing the space with empty string.

**Comment by aksg87** [comment]
> I dont see a point using yaml;
> also the code makes the csv file.

@Florassss  I think it's best practice to use YAML or JSON. In this case it's not a big issue but easy enough to make the change. 

You can see this discussion which goes into it a bit more: https://softwareengineering.stackexchange.com/questions/351126/how-bad-of-an-idea-is-it-to-use-python-files-as-configuration-files

**Comment by aksg87** [comment]
> > With `pathlib.Path` you can just say `filename.parent` which is more clear I think!
> 
> The reason I couldn't use pathlib for new path is the sub dir has space in the string. that's why I am replacing the space with empty string.

pathlib.Path can handle paths with spaces I think? I just tested this with a simple example:

```
In [1]: !ls
'test pathlib'

In [2]: from pathlib import Path

In [3]: path = Path('test pathlib')

In [4]: path.exists()
Out[4]: True
```

File: deployment/pipeline.py [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+
+class Handler(FileSystemEventHandler):
+    @staticmethod
+    def on_any_event(event):
+        if event.is_directory:
+            return None
+        elif event.event_type == "created":
+            # Event is created, can process it now
+            print("Watchdog received created event - % s." % event.src_path)
+            filename = event.src_path.split("/")[-1]
```
Comment (aksg87) [inline deployment/pipeline.py:194]: `filename = event.src_path.split("/")[-1]`
-->
`filename = Path(event.src_path)"

**Pull request closed without merging**